Source/WebKit/UIProcess/API/Cocoa/WKWebViewPrivate.h

```diff
@@ -216,6 +216,7 @@ typedef NS_ENUM(NSInteger, _WKImmediateActionType) {
 @property (nonatomic, readonly) pid_t _modelProcessIdentifier WK_API_AVAILABLE(macos(15.0), ios(18.0), visionos(2.0));
 
 @property (nonatomic, getter=_isEditable, setter=_setEditable:) BOOL _editable WK_API_AVAILABLE(macos(10.11), ios(9.0));
+@property (nonatomic, setter=_setShowsColorInToolbar:) BOOL _showsColorInToolbar WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA));
```

Comment:
I think this name is a bit too vague (and also assumes that the top content inset area is underneath a toolbar). The color in question is also the background of a view underneath the toolbar (rather than in Safari's toolbar).

Perhaps something like `_shouldSuppressTopFixedColorExtensionView`?